apps/opik-backend/src/test/java/com/comet/opik/api/resources/v1/internal/UsageResourceTest.java

```diff
@@ -56,7 +70,7 @@ public class UsageResourceTest {
     private static final MySQLContainer<?> MYSQL_CONTAINER = MySQLContainerUtils.newMySQLContainer();
 
     private static final ClickHouseContainer CLICK_HOUSE_CONTAINER = ClickHouseContainerUtils
-            .newClickHouseContainer(false);
+            .newClickHouseContainer();
```

Comment:
If you remove this, the test will share the container with other tests, can some test will fail locally

Reply:
It's ok. Each test creates, modifies and checks it's own data, so there should be no problems.